src/packages/cli/src/bin.ts

```diff
@@ -189,14 +189,27 @@ async function main(): Promise<number> {
     let schemaProvider: string | undefined
     let schemaPreviewFeatures: string[] | undefined
     let schemaGeneratorsProviders: string[] | undefined
+
     try {
       const schema = await getSchema(args['--schema'])
       const config = await getConfig({
         datamodel: schema,
+        ignoreEnvVarErrors: true,
```

Comment:
Looks like we forgot to so that a long time ago, without it was throwing when env var was not set for example.

Reply:
Done in https://github.com/prisma/prisma/pull/8529